x-pack/plugins/apm/public/components/app/service_groups/service_groups_tour.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiSpacer, EuiText, EuiTourStep } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { ElasticDocsLink } from '../../shared/links/elastic_docs_link';
+
+export type TourType = 'createGroup' | 'editGroup' | 'serviceGroupCard';
+
+interface Props {
+  title: string;
+  content: string;
+  tourEnabled: boolean;
+  dismissTour: () => void;
+  children: React.ReactElement;
+}
+
+export function ServiceGroupsTour({
+  tourEnabled,
+  dismissTour,
+  title,
+  content,
+  children,
+}: Props) {
+  return (
+    <EuiTourStep
+      content={
+        <>
+          <EuiText size="s" color="subdued">
+            {content}
+          </EuiText>
+          <EuiSpacer />
+          <ElasticDocsLink
+            section="/kibana"
+            path="/services.html"
+            target="blank"
+          >
+            <EuiText size="s" color="subdued">
+              {i18n.translate('xpack.apm.serviceGroups.tour.content.link', {
+                defaultMessage: 'Learn more in our documentation',
+              })}
+            </EuiText>
+          </ElasticDocsLink>
```

Comment:
I think we'd actually want these docs links to be formatted like this (visual).

![localhost_5601_miw_app_apm_services_environment=ENVIRONMENT_ALL rangeFrom=now-15m rangeTo=now comparisonEnabled=true comparisonType=day (1)](https://user-images.githubusercontent.com/4104278/159430139-63ca265f-1f65-4232-9976-389c79659697.png)

Reply:
<img width="373" alt="Screen Shot 2022-03-22 at 2 46 10 PM" src="https://user-images.githubusercontent.com/55978943/159553915-0dec5f4e-3531-4b95-bd79-716ce8cccea5.png">